views/components/form/resource_creation_form.erb

```diff
@@ -104,8 +104,12 @@ form_elements.each do |element|
       </div>
       <div class="px-4 py-5 sm:p-6">
         <div class="flex items-center justify-end gap-x-6">
-          <a href="<%= action %>" class="text-sm font-semibold leading-6 text-gray-900">Cancel</a>
-          <%== part("components/form/submit_button", text: "Create") %>
+          <% if mode == "create" %>
+            <a href="<%= action %>" class="text-sm font-semibold leading-6 text-gray-900">Cancel</a>
```

Comment:
The update mode also might have a cancel button. Imagine you select different values for some properties but then decide not to apply them. The cancel button can refresh the page and load the current values.